zebrad/src/components/sync.rs

```diff
@@ -21,13 +21,16 @@ use zebra_state as zs;
 use crate::{config::ZebradConfig, BoxError};
 
 mod downloads;
+mod gossip;
```

Comment:
What about changing this into `gossip_blocks`? we are going to add a similar module named `gossip_transactions` for https://github.com/ZcashFoundation/zebra/issues/2758

Reply:
I was thinking of something like:
* blocks: `components::sync::gossip`
* transactions: `components::mempool::gossip`

The syncer downloads, verifies, and gossips blocks.
The mempool downloads, verifies, and gossips unmined transactions.